monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/RunManually/LocalManualRunOptions.js

```diff
@@ -72,6 +84,7 @@ const getContents = (props) => {
         </div>
       </div>
       <CommandDisplay commands={commands}/>
+      <Button style={{'float': 'right'}} title="Re-generate run monkey command" onClick={getOtp}>Re-generate</Button>
```

Comment:
I think I'd prefer this to be "Refresh" instead of "Re-generate". Maybe even "Refresh OTP". I'm open to thoughts from the group.

Reply:
<img width="1158" alt="image" src="https://user-images.githubusercontent.com/15820737/230307349-b02185db-11ca-44c1-89f7-72c7beb9b7f1.png">